packages/dom/src/text-quote/describe.ts

```diff
@@ -21,44 +21,31 @@
 import seek from 'dom-seek';
 import type { TextQuoteSelector } from '@annotator/selector';
 
-import type { DomScope } from '../types';
-import { ownerDocument, rangeFromScope } from '../scope';
-
 export async function describeTextQuote(
   range: Range,
-  scope: DomScope = ownerDocument(range).documentElement,
+  scope: Range = range,
```

Comment:
Should be the document element, not the range itself!

Reply:
I changed it so that it took a range, because it simplified things and made this symmetric with the anchoring.

Reply:
Oh, my mistake, I am thinking of the other function to calculate context.